Makefile.toml

```diff
@@ -0,0 +1,107 @@
+[env]
+CARGO_MAKE_EXTEND_WORKSPACE_MAKEFILE = "true"
+CARGO_MAKE_WORKSPACE_EMULATION = true
+CARGO_MAKE_CRATE_WORKSPACE_MEMBERS = [
+	"crt0stack",
+	"enarx-keep-sgx",
+	"enarx-keep-sgx-shim",
+	"enumerate",
+	"intel-types",
+	"iocuddle",
+	"iocuddle-sgx",
+	"memory",
+	"nolibc",
+	"payload",
+	"sev",
+	"sev-show",
+	"sev-types",
+	"sgx-crypto",
+	"sgx-show",
+	"sgx-types",
+	"span",
+	"testing",
+	"units",
+	"vdso",
+	"vmsyscall"
+]
+
+[tasks.audit]
+install_crate = "cargo-audit"
+command = "cargo"
+args = ["audit"]
+
+[tasks.deny]
+install_crate = "cargo-deny"
+command = "cargo"
+args = ["deny", "check", "licenses"]
+
+[tasks.test]
+command = "cargo"
+args = ["test"]
+
+[tasks.check-format]
+command = "cargo"
+args = ["fmt", "--", "--check"]
```

Comment:
`install_crate = "cargo-fmt"`?

Reply:
Is there a reason we can't use this implicitly? https://github.com/sagiegurari/cargo-make/blob/master/src/lib/descriptor/makefiles/rust.toml#L205

Reply:
I think plain `cargo fmt` changes the source files and then returns `0` as its exit code which would allow our CI test to pass even if the source code is not properly formatted. `cargo fmt --check` will exit with a non-zero code which will be noticed by CI.

Reply:
@connorkuehl Since upstream is so active, we should add a `--check` version of that to their built-in test flow. That way:
1. We don't have to maintain it.
2. Others can benefit from it.

However, I'm willing to take this as a `debt` issue to keep our momentum.

Reply:
This is my mistake again, there is indeed a builtin for this. However, the `install_crate` step is gated behind nightly (which used to be the case before `rustfmt` was included in the stable toolchain). I've [filed an issue upstream](https://github.com/sagiegurari/cargo-make/issues/418) to remove this reqiurement.

Reply:
I'm aware it is installed on recent rust toolchain installs. But we might have old toolchain installs or even people who have removed it.